### PR TITLE
Upgrade to Python 3.11

### PR DESCRIPTION
**Pull request by shifucun**
A few things updated together:

* Use light-the-torch to install cpu only torch
* Install en_core_web_lg with "-f" option, so no need to do separate check for en_core_web_lg
* Disable googlecloudprofiler as it has issue with the latest wheel setup. This is not used now.

### CODE REVIEW

File: server/__init__.py [hunk 1]
```diff
@@ -219,18 +219,18 @@ def register_routes_common(app):
...
-
-    # Profiler initialization. It starts a daemon thread which continuously
-    # collects and uploads profiles. Best done as early as possible.
-    try:
-      # service and service_version can be automatically inferred when
-      # running on GCP.
-      googlecloudprofiler.start(verbose=3)
-    except (ValueError, NotImplementedError) as exc:
-      logging.error(exc)
+  # if os.environ.get('FLASK_ENV') in ['production', 'staging', 'autopush']:
```
Comment (chejennifer) [inline server/__init__.py:222]: should we just delete this if cloudprofiler isn't being used?

File: server/routes/place/html.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 
 @bp.route('', strict_slashes=False)
-@bp.route('/<path:place_dcid>', strict_slashes=False)
+@bp.route('/<path:place_dcid>/', strict_slashes=False)
```
Comment (chejennifer) [inline server/routes/place/html.py:33]: curious why we need the extra slash

**Review by chejennifer — APPROVED** [review]
thanks for upgrading!

File: server/__init__.py (hunk 1 above)
Comment (shifucun) [reply server/__init__.py]: Removed

File: server/routes/place/html.py (hunk 2 above)
Comment (shifucun) [reply server/routes/place/html.py:33]: The latest Flask can not handle the trailing slash without specifying it explicitly.

**Review by pradh — APPROVED** [review]
Thanks a lot for doing this!

Very exciting!

**Review by jehangiramjad — APPROVED** [review]
LGTM! Many thanks :)

**Review by jehangiramjad — APPROVED** [review]
LGTM! Many thanks :)

**Pull request merged**